src/com/twitter/intellij/pants/service/PantsCompileOptionsExecutor.java

```diff
@@ -169,12 +169,12 @@ private String loadProjectStructureFromTargets(
   ) throws IOException, ExecutionException {
     final File outputFile = FileUtil.createTempFile("pants_depmap_run", ".out");
     final GeneralCommandLine command = getCommand(outputFile, statusConsumer);
-    statusConsumer.consume("Resolving dependencies...");
+    statusConsumer.consume("Exporting Pants project into IDEA format...");
```

Comment:
it's fine to keep the old one, and it will ease the documentation load.

Reply:
reverted